setup.sh

```diff
@@ -46,7 +46,32 @@ XCODE_VERSION=( ${version//./ } )
 rm -rf ${TMPDIR}/TemporaryItems/*carthage*
 
 echo "ℹ️  Carthage bootstrap. This might take a while..."
-carthage bootstrap --cache-builds --platform ios --use-xcframeworks
+if [[ -z "${CIRRUS_BUILD_ID}" ]]; then # skip cache bootstrap for CI
+    carthage bootstrap --cache-builds --platform ios --use-xcframeworks
+else 
+    echo "Skipping Carthage bootstrap from setup.sh script since it's running on Cirrus-CI"
+fi 
+echo ""
+
+echo "ℹ️  Installing ImageMagick..."
+if [[ -z "${CIRRUS_BUILD_ID}" ]]; then # skip cache bootstrap for CI
+    echo "Skipping ImageMagick install because not running on Cirrus-CI"
+else
+    which identify || brew install ImageMagick
+fi 
```

Comment:
What are we using this for? To add build numbers to the app icon?

Reply:
Yes 😄